### ISSUE TITLE
Question: Is there a way to configure underlying jsdoc?

### ISSUE THREAD

**Issue by daserge** [jsdoc2md/jsdoc-to-markdown#45]
`jsdoc` allows to pass config path like this:

```
jsdoc www -r -c \"jsdoc2md/conf.json\" -d docs
```

Is there a way to do this with `jsdoc-to-markdown` (so that @partial plugin can be used for example)?

**Comment by 75lb** [comment]
not yet, but it is on my list of things to do.. i will get back to you Sergey!

**Comment by daserge** [comment]
@75lb, got it, thanks for the quick response!

By the way - is there any workaround for this for now?
Do I get it right that Handlebars templates will not allow to wedge into jsdoc custom tags like @ partial does?

**Comment by 75lb** [comment]
what are you trying to achieve? I'm sure there's a different, maybe better way, than using @partial.

**Comment by daserge** [comment]
I need to insert blocks of documentation, which are too big to include them in code, while are still related to the code.

**Comment by 75lb** [comment]
if i remember correctly it's possible to document an identifier in a separate file..the @typedef tag is documented separately this way - not attached to code.. 

so if you had a separate file (e.g. `docs.js`) containing only doclets, like these
```
/**
@function move
@memberof Train
*/

/**
@member size
@type {string}
@memberof Train
*/

```

then you could include this file in the list of input files to parse
```
$ jsdoc2md lib/*.js
```

would this solve your problem? I have never tried it.. this is just a quick idea of the top of the head..

**Comment by daserge** [comment]
@75lb, thanks a lot for the workaround!
I don't think it will fit unfortunately as the large docs chunks are meant to be Markdown, i.e. they have to be editable/previewed on Github.

**Comment by 75lb** [comment]
released in jsdoc2md v1.3.0 - see the `--conf` option
